toolkit/scripts/toolchain/create_toolchain_in_container.sh

```diff
@@ -18,9 +18,6 @@ docker ps -a
 # docker rmi $(docker images -a -q)
 # docker rmi $(docker history marinertoolchain -q)
 
-# OpenJDK8 patch files
-cp -v $MARINER_SPECS_DIR/openjdk8/Awt_build_headless_only.patch ./container
```

Comment:
I believe these files have corresponding entries in `toolkit/scripts/.gitignore`, can we remove those as well?

Reply:
Sure thing - will remove